src/request.ts

```diff
@@ -516,7 +523,7 @@ class DatastoreRequest {
   }
 
   insert(entities: Entities): Promise<CommitResponse>;
-  insert(entities: Entities, callback: CallOptions): void;
+  insert(entities: Entities, callback: CommitCallback): void;
```

Comment:
Should we follow suit with the other commit convenience methods and make a custom type for the callback/response here that extends the commit version?

Reply:
I think yes, nice catch!
Added `InsertCallback`/`Response` and `DeleteCallback`/`Response`